cmd/repl/prompt.go

```diff
@@ -327,7 +326,7 @@ func (p *promptState) navigateHistory(origInput string, older bool, buf *prompt.
 		}
 	}
 	prompt.GoLineBeginning(buf)
-	buf.Delete(len(buf.Text()))
+	buf.Delete(len([]rune(buf.Text())))
```

Comment:
Is it possible to add a unit test for this case?

Reply:
Yes, I will add one.